spec/vcr/services/controlled_terms/genre_canonicable.yml

```diff
@@ -0,0 +1,121 @@
+---
+http_interactions:
+- request:
```

Comment:
This is a recording of a bad request (404 returned). Do we need this? Appears in a bunch of the `*_cannonicable.yml` VCR files.